tests/bluetooth/bsim_bt/bsim_test_advx/src/main.c

```diff
@@ -105,6 +199,70 @@ static void test_advx_main(void)
 
 	k_sleep(K_MSEC(1000));
 
+	printk("Create advertising set...");
+	err = bt_le_ext_adv_create(BT_LE_ADV_CONN_NAME, &adv_callbacks, &adv);
+	if (err) {
+		goto exit;
+	}
+	printk("success.\n");
+
+	printk("Start advertising using extended commands (max_events)...");
+	is_sent = false;
+	ext_adv_param.timeout = 0;
+	ext_adv_param.num_events = 3;
+	err = bt_le_ext_adv_start(adv, &ext_adv_param);
+	if (err) {
+		goto exit;
+	}
+	printk("success.\n");
+
+	printk("Waiting...");
+	while (is_sent) {
+		k_sleep(K_MSEC(100));
+	}
+	printk("done.\n");
+
+	k_sleep(K_MSEC(1000));
+
+	printk("Start advertising using extended commands (duration)...");
+	is_sent = false;
+	ext_adv_param.timeout = 50;
+	ext_adv_param.num_events = 0;
+	err = bt_le_ext_adv_start(adv, &ext_adv_param);
+	if (err) {
+		goto exit;
+	}
+	printk("success.\n");
+
+	printk("Waiting...");
+	while (is_sent) {
+		k_sleep(K_MSEC(100));
+	}
+	printk("done.\n");
+
+	k_sleep(K_MSEC(1000));
+
+	printk("Start advertising using extended commands (disable)...");
+	is_sent = false;
+	ext_adv_param.timeout = 0;
+	ext_adv_param.num_events = 5;
+	err = bt_le_ext_adv_start(adv, &ext_adv_param);
+	if (err) {
+		goto exit;
+	}
+	printk("success.\n");
+
+	k_sleep(K_MSEC(400));
+
+	printk("Stopping advertising using extended commands...");
+	err = bt_le_ext_adv_stop(adv);
+	if (err) {
+		goto exit;
+	}
+	printk("success.\n");
+
+	k_sleep(K_MSEC(1000));
```

Comment:
We need to verify that advertising indeed has stopped, and then this sleep is not necessary

Reply:
The sleep is added to have clear differentiation (gaps) when analyzing exported Ellisys traces.